src/types/user.ts

```diff
@@ -11,7 +11,7 @@ export interface IUser {
   agency: IAgencySchema['_id']
   contact?: string
   created?: Date
-  betaFlag?: {}
+  betaFlag?: Record<string, unknown>
```

Comment:
why is this unknown?

Reply:
Set it to `never` instead, since it should not have any members now.